README.md

```diff
@@ -17,7 +17,24 @@ kubectl create namespace sensrnet-registry
 ```
 
 ### TCP traffic for multichain node
-One of the components, the multichain node, requires an TCP ingress. This is not natively supported by Kubernetes. Extra accomodation have to be made in the Loadbalancer to make the node accessible from outside, as regular HTTP connections won't work. Two flavours of Loadbalancers have been tested, Traefik and Nginx-ingress.
+One of the components, the multichain node, requires an TCP exposure. By default this helm chart will expose on the internal load balancer only (a.k.a. `ClusterIP`) and external exposure needs to be configured properly with flags. There are three options available (and tested):
+
+- Kubernetes Service Type `LoadBalancer`
+- Traefik Ingress Controller
+- Nginx Ingress Controller
+
+Please apply one of these which is most applicable to your situation.
+
+#### Kubernetes Service Type `LoadBalancer`
+
+It is possible to deploy a Kubernetes service on an external available load balancer. In the helm charts the service for MultiChain has the appropriate TCP port configured. By default the service type is set to `ClusterIP` and this has to be updated to `LoadBalancer`:
```

Comment:
Perfect description! I would only add a small warning that using this route might come with additional costs for the hosting of the Kubernetes cluster

Reply:
Done! Thnx for the comments